packages/back-end/src/services/context.ts

```diff
@@ -78,7 +78,7 @@ export class ReqContextClass {
   }
 
   public org: OrganizationInterface;
-  public usage: () => Promise<OrganizationUsage>;
+  public usage: OrganizationUsage;
```

Comment:
Should this property still be here?  It doesn't look like it's being initialized anywhere